hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/dag/nodes/CleanNode.java

```diff
@@ -32,7 +32,7 @@ public CleanNode(Config config) {
   }
 
   @Override
-  public void execute(ExecutionContext executionContext) throws Exception {
+  public void execute(ExecutionContext executionContext, int itrCount) throws Exception {
```

Comment:
Can we just use "REPEAT_COUNT" ?

Reply:
as synced up offline, this refers to current iteration count. Have renamed this arg to "curItrCount".